### ISSUE TITLE
machine-to-machine

### ISSUE THREAD

**Issue by sparrell** [oasis-tcs/openc2-oc2ls#173]
This is a child of #143 

```
Page 2, Line 7 (and elsewhere):Â machine-to-machine should be hyphenated pretty much anywhere it appears in this document.


```

**Comment by jmbrule** [comment]
I got comments both ways (should hyphenate, should not ) and did a google and saw it both ways.   Currently is not hyphenated in the SLPF, but easy fix if we decide to hyphenate.  When this is resolved, need to ensure all three specks are consistent

**Comment by jmbrule** [comment]
OK, are we in agreement that it is hyphenated for all three specks?  If so, will fix in the SLPF and close

**Comment by romanojd** [comment]
I've addressed this issue within issue #143. Closing this comment.
